src/components/multiStep/index.js

```diff
@@ -82,6 +82,7 @@ class MultiStep extends React.Component {
       prevStep: step.prevStep,
       reset: this.reset.bind(this),
       ...step.data[step.current],
+      finalCallback,
```

Comment:
If you add `finalCallback` here, the condition `if (step.current === (children.length - 1)) {` below is not needed

Reply:
you are right it is fixed now